hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/utils/db/TestDBConfigFromFile.java

```diff
@@ -90,23 +70,21 @@ public void readFromFile() throws IOException {
 
   @Test
   public void readFromFileInvalidConfig() throws IOException {
-    final List<String> families =
-        Arrays.asList(StringUtils.bytes2String(RocksDB.DEFAULT_COLUMN_FAMILY),
-            "First", "Second", "Third",
-            "Fourth", "Fifth",
-            "Sixth");
-    final List<ColumnFamilyDescriptor> columnFamilyDescriptors =
-        new ArrayList<>();
-    for (String family : families) {
-      columnFamilyDescriptors.add(
-          new ColumnFamilyDescriptor(family.getBytes(StandardCharsets.UTF_8),
-              new ColumnFamilyOptions()));
-    }
-
-    final DBOptions options = DBConfigFromFile.readFromFile("badfile.db.ini",
-        columnFamilyDescriptors);
-
+    final DBOptions options = DBConfigFromFile.readDBOptionsFromFile(Paths.get("badfile.db.ini"));
     // This has to return a Null, since we have config defined for badfile.db
     assertNull(options);
   }
+
+  @Test
```

Comment:
Please also check that edge cases (such as an empty or misconfigured ini file) are covered in the test suite.